Source/Charts/Renderers/CandleStickChartRenderer.swift

```diff
@@ -38,9 +38,9 @@ open class CandleStickChartRenderer: LineScatterCandleRadarRenderer
             accessibleChartElements.append(element)
         }
 
-        for case let set as CandleChartDataSetProtocol in candleData where set.isVisible
+        for case let set as CandleChartDataSetProtocol in (candleData as ChartData) where set.isVisible
```

Comment:
set is not useful name for it, because set is special word in swift )

Reply:
I changed to :
`for case let dataSet as CandleChartDataSetProtocol in (candleData as ChartData) where dataSet.isVisible`